.gitignore

```diff
@@ -48,6 +48,7 @@ autoconf/autom4te.cache
 # (Rust) Cargo lockfile, not published with libraries.
 Cargo.lock
 bindings/rust/target
+/target
```

Comment:
Rust crates are moved into folder `bindings/rust`, so the `/target` will not be generated.  
Is this `target` folder still generated in your environment?

Reply:
Because I used the previous version of the code, I forgot to change this place. I will remove it when I officially pr.